src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/envelopes/EnvelopeEventProcessor.java

```diff
@@ -196,6 +200,24 @@ private void logMessageParsed(IMessage message, Envelope envelope) {
         );
     }
 
+    private void logFeignException(
```

Comment:
you could reuse `logMessageProcessingError` only the base message will change if it is Feign Exception